401-installation-state.md

```diff
@@ -0,0 +1,445 @@
+---
+title: Installation State
+weight: 401
+---
+
+# CNAB Installation State 1.0.0
+*Draft, Mar. 2021*
+
+This specification describes CNAB Installation State. This is not part of the [CNAB Core](100-CNAB.md) specification, but is a specification describing how state related to an installation of a bundle is formatted and stored. Implementations of CNAB Core can meet the CNAB Core specification without implementing this specification. Implementations that support installation state MAY declare that they comply with CNAB Installation State 1.0.0.
+
+The Installation State specification is based on the CNAB Claims specification, extending it to cover additional data that MAY be used by a CNAB runtime. After we have implemented and vetted this specification, one possible outcome would be to replace the Claims specification with this one.
+
+A _host environment_ is an environment, possibly shared between multiple CNAB tools, that provides persistence for CNAB documents. For example, a filesystem may contain a record of claims that two different CNAB clients share. Or a database may contain the environment that is shared by multiple tools at different locations in the network. Each of these is a CNAB host environment.
```

Comment:
This has been the term since the beginning of the CNAB spec (it's taken from the claims spec). I also think it is unclear and am happy to come up with a different name for it, as long as we understand that changing the name could lead to two different specs with different terms for the same thing.

Reply:
Also the host environment can contain more than just installation state data. It also has credential/parameter stores, and potentially other documents used to the cnab tool that aren't covered in the spec.

Reply:
I'd forgotten that we used it elsewhere. You're right, we should be consistent and if it isn't clear we can discuss it as a separate issue.